druid-shell/src/platform/windows/keyboard.rs

```diff
@@ -608,7 +608,7 @@ impl KeyboardState {
                     let scan_code = ((lparam & SCAN_MASK) >> 16) as u32;
                     let vk = self.refine_vk(stash_vk.unwrap_or(0), scan_code);
                     let code = scan_to_code(scan_code);
-                    let key = if self.stash_utf16.is_empty() && wparam < 0x20 {
+                    let key = if self.stash_utf16.is_empty() && (wparam < 0x20 || wparam == 0x7f) {
```

Comment:
This was already commited to master, not sure why it appears here again, I might need to do a new pull request to remove this line.